src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj

```diff
@@ -26,7 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.AWS.Tests" />
```

Comment:
What do we use internals for?

Reply:
![image](https://github.com/user-attachments/assets/1ee9c050-34b5-420f-b63e-2e190f1c9eda)